tests-behat/bootstrap/Context.php

```diff
@@ -452,23 +444,20 @@ protected function getFinishedScript(): string
     {
         return 'document.readyState === \'complete\''
             . ' && typeof jQuery !== \'undefined\' && jQuery.active === 0'
-            . ' && typeof atk !== \'undefined\' && atk.vueService.areComponentsLoaded()'
-            . ' && jQuery(\':animated\').length === 0'; // needed for SUI dropdown
```

Comment:
no `:animated` or `.loading` class condition needed, which is great, as these can slow down this check